network/setup-ip

```diff
@@ -17,20 +25,24 @@ configure_network() {
     local secondary_dns="${10}"
 
     /sbin/ifconfig "$INTERFACE" "$ip" netmask "$netmask"
+    /sbin/ip -- neighbour replace to "$ip" dev "$INTERFACE" \
+        lladdr "$MAC" nud permanent
```

Comment:
This adds VM's own IP+MAC to the neighbour table, I don't think you meant that. It should be rather gateway's IP+MAC. In theory there is an issue with that - you don't have gateway's MAC in this script. But in practice, _with Xen_, vif interface always has `fe:ff:ff:ff:ff:ff` MAC, at least on Linux (requires checking in MirageOS).

@hannesm can you confirm/deny if the above about MAC address on netback is true on MirageOS too?

Reply:
Thanks for the mention @marmarek. I think this is the case according to https://github.com/mirage/mirage-net-xen/pull/87 and https://github.com/mirage/qubes-mirage-firewall/pull/61 //cc @talex5.